andela/hc-wits-kla
# [Feature #158175314] Render Failed Jobs Tab

#### What does this PR do?
* Renders an Authorized User's Unresolved Checks

#### Description of Task to be completed?
* Add tab to the Base User Page
* Create Page for Failing Checks ([Feature #158175314] Add Failed check tab and custom page)
* Retrieve all the Authorized User's Failing checks ([Feature #158175314] Render failed checks on the unresolved stuff tab)

#### How should this be manually tested?
- Clone the project from GitHub ```git clone https://github.com/andela/hc-wits-kla.git```
- After cloning, cd into the project directory and run the environment setup in the project README
- Run the project URL from the terminal console in your browser
- Create an Account and Login
- Add some checks, ping some of the checks, adjust the period and grace period to a lower time to have the check drop
- When the check drops, an alert is sent to the user and the check is added to the Failed Checks Tab

#### Any background context you want to provide?
- When the difference between the time now and the and a summation of the check's last ping, period and grace periods is positive then the check is down and an email alert is sent to the owner.

#### What are the relevant pivotal tracker stories?
* #158175314

#### Screenshots

MY CHECKS SCREENSHOT

<img width="1289" alt="screen shot 2018-06-27 at 16 28 03" src="https://user-images.githubusercontent.com/39955231/41977257-8e322796-7a27-11e8-967f-ee7a0a7d6c89.png">

MY FAILED CHECKS SCREENSHOT

<img width="1362" alt="screen shot 2018-06-27 at 16 28 23" src="https://user-images.githubusercontent.com/39955231/41977318-ae44a9e6-7a27-11e8-9114-39ff4ad206d9.png">
